pyccel/ast/numpyext.py

```diff
@@ -544,32 +544,83 @@ def Shape(arg):
 class NumpyLinspace(NumpyNewArray):
 
     """
-    Represents numpy.linspace.
+    Represents numpy.linspace which returns num evenly spaced samples, calculated over the interval [start, stop].
 
+    Parameters
+      ----------
+      start           : list , tuple , PythonTuple, PythonList, Variable, Literals
+                        Represents the starting value of the sequence.
+      stop            : list , tuple , PythonTuple, PythonList, Variable, Literals
+                        Represents the ending value of the sequence (if endpoint is set to False).
+      num             : int, optional
+                        Number of samples to generate. Default is 50. Must be non-negative.
+      endpoint        : bool, optional
+                        If True, stop is the last sample. Otherwise, it is not included. Default is True.
+      dtype           : str, DataType
+                        The type of the output array. If dtype is not given, the data type is calculated
+                        from start and stop, the calculated dtype will never be an integer.
```

Comment:
The dtype argument needs testing

Reply:
i tested it manually, I need just to add the tests to the file

Reply:
As @EmilyBourne pointed out, the `dtype` argument is not tested yet. Can you please add some basic checks?